src/engine/source/metrics/input_files/metrics-config.json

```diff
@@ -0,0 +1,56 @@
+[
+    {
+        "name": "CountExample",
+        "enable": true,
+        "signalType": "meter",
+        "instrumentType" : "counter",
+        "subType": "uint64",
+        "outputFile": "countExample.txt",
+        "exportIntervalMillis": 1000,
+        "exportTimeoutMillis": 500
+    },
+    {
+        "name": "HistogramExample",
+        "enable": true,
+        "signalType": "meter",
+        "instrumentType" : "histogram",
+        "subType": "double",
+        "outputFile": "histogramExample.txt",
+        "exportIntervalMillis": 1000,
+        "exportTimeoutMillis": 500
+    },
```

Comment:
Add the "boundaries" element to save the frecuency of histogram.
Add the "unit" to determinate the unit of metric values. For example: unit: "milliseconds".

See https://opentelemetry-cpp.readthedocs.io/en/latest/otel_docs/classopentelemetry_1_1metrics_1_1Meter.html and https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#histogram